desktop/cmp/input/JsonInput.ts

```diff
@@ -25,6 +25,7 @@ export const [JsonInput, jsonInput] = hoistCmp.withFactory<JsonInputProps>({
             linter: linter,
             formatter: fmtJson,
             mode: 'application/json',
+            testId: props.testId,
```

Comment:
I don't think this is necessary since the props are being spread on the next line anyway